recipes/lsp-cfn

```diff
@@ -0,0 +1,4 @@
+(lsp-cfn
+ :repo "LaurenceWarne/lsp-cfn.el"
+ :fetcher github
+ :files ("*.el" "snippets"))
```

Comment:
Unless there's a good reason not to --
```suggestion
 :files (:defaults "snippets"))
```

Reply:
Thanks, sorry I missed this from the guidance